### DISCUSSION TITLE
About several users (for access to web GUI).

### DISCUSSION THREAD

**Discussion by Yuri-01** [AdguardTeam/AdGuardHome#3129] [Q&A]
Hi.

I see that i can create multiple users (for access to Web-mgmt GUI).
What for exist this capability?
(after all, all users still have equal rights to manage "AdGuard Home", but "AdGuard Home" does not have any role-based access features).

**Comment by Aikatsui** [comment]
See https://github.com/AdguardTeam/AdGuardHome/issues/997

**Reply by Yuri-01** [reply]
Thank you.
